test/addons-napi/3_callbacks/test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const common = require('../../common');
+const assert = require('assert');
+const addon = require(`./build/${common.buildType}/binding`);
+
+addon.RunCallback(function(msg) {
+  assert.strictEqual(msg, 'hello world');
+});
+
+const global = function() { return this; }.apply();
```

Comment:
Huh? This is strict mode, if you want the global object, just leave `global` the way it is.